pkg/apis/triggers/v1beta1/trigger_types.go

```diff
@@ -166,7 +166,13 @@ type BitbucketInterceptor struct {
 type GitHubInterceptor struct {
 	SecretRef *SecretRef `json:"secretRef,omitempty"`
 	// +listType=atomic
-	EventTypes []string `json:"eventTypes,omitempty"`
+	EventTypes      []string              `json:"eventTypes,omitempty"`
+	AddChangedFiles GithubAddChangedFiles `json:"addChangedFiles,omitempty"`
```

Comment:
Hmm...we used to have these structs in the trigger_types package since we used to support specifying `secretRef `and `eventTypes` as fields vs the param/value based approach that we do now. I wonder if it is possible to move this struct into the specific interceptor implementation itself. I think doing so would mean that newly added params to an interceptor don't show up in the `triggers-api.md` file

Reply:
@dibyom do you mean moving `AddChangedFiles` to specific Interceptor i.e Github Interceptor  and keeping `secretRef` and `eventTypes` as it is in trigger_types.go in order to make sure if there are anything specific to Interceptor will be added to that respective Interceptor ?

Reply:
yes, that is the intention. If that is too complicated to do in this PR, we could open a clean issue to track this

Reply:
+1 for doing it as another PR 
WDYT @kcloutie

Reply:
@dibyom / @savitaashture  I looked at the change and it doesn't appear to be very easy. Moving only `AddChangedFiles` struct will cause a circular reference and would require, at the very least, to move `GitHubInterceptor ` struct.. I vote for a new issue to do that change. Let me know if you agree

Reply:
Yes create a new issue and that you can handle in new PR